src/main/java/org/jenkinsci/plugins/workflow/libs/LibrariesAction.java

```diff
@@ -33,11 +33,14 @@
 import java.io.IOException;
 import java.util.Collections;
 import java.util.List;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
 
 /**
  * A run action recording libraries used in a given build.
  */
-class LibrariesAction extends InvisibleAction {
+@ExportedBean
+public class LibrariesAction extends InvisibleAction {
```

Comment:
This class was not previously public. I could add `@Restricted(NoExternalUse.class)` to prevent direct Java-level access from other plugins if desired, but the class seems pretty harmless and likely to be stable going forward so it seemed ok to me to just make it public.